config/jobs/kubernetes/sig-api-machinery/compatibility-versions-e2e.yaml

```diff
@@ -3,7 +3,7 @@ periodics:
   cluster: k8s-infra-prow-build
   name: ci-kubernetes-e2e-kind-compatibility-versions-n-minus-1
   annotations:
-    testgrid-dashboards: sig-testing-kind
+    testgrid-dashboards: sig-api-machinery-emulated-version
```

Comment:
aside: we should probably decide on some consistent naming, right now we have at least three ways to refer to this:
- emulation version
- compatbility version
- emulated version

Reply:
I used emulated version since that is the name for the flag, and I believe we are phasing out compatibility version entirely. Not sure between emulation and emulated though.